challenges/platform/README.md

```diff
@@ -0,0 +1,59 @@
+# Summary
+
+Teleport Cloud provides secure access to servers, databases, and applications of our customers. The platform team is responsible for automating, scaling, monitoring, and securing the infrastructure that underpins Teleport Cloud. This team draws on a wide variety of skills and experience to build reliable infra, and as such we use different interviews based on the candidates background.
+
+
+# Rationale
+
+These exercises have two goals.
+
+- It helps us understand what to expect from you as an engineer, how you reason about production systems and investigate faults.
+- It helps you get a feel for what it would be like to work at Teleport, as this exercise aims to simulate some of our day-as-usual and expose you to the type of work we're doing here.
+
+We believe this technique is better and is more fun compared to whiteboard/quiz interviews so common in the industry. It's not without the downsides as it could take longer than traditional interviews. [Some of the best teams use coding challenges.](https://sockpuppet.org/blog/2015/03/06/the-hiring-post/)
+
+We appreciate your time and are looking forward to hacking on this challenge together.
+
+
+# Backgrounds
+
+## Systems / Backend Developers
+Systems developers on the platform team leverage a strong background in software development to build, maintain, deploy core production services.
+
+### Challenge
+1. [Systems developer challenge](../systems/worker.pdf)
+
+### Levels
+See [levels.pdf](../../levels.pdf)
+
+
+## Devops / DevSecOps / System Administrators / Production engineers / etc
```

Comment:
s/System Administrators/SREs/ ?

My understanding is we're not presently looking to hire anyone _seeking_ a System Administrator role, but it is absolutely great to have folks with sysadmin in their background if they're seeking for a devops/infrastructure as code role.

Reply:
Yea, it's really tricky, oldschool sysadmins probably did all of these requirements and more, but got a bad reputation, so we starting calling them devops, which now has just as much of a bad reputation as companies started calling their admin teams DevOps, so now they're called SREs, and now SREs have a bad rep in some circles..

Reply:
I tried to target these more to focus areas, and then include the example list. LMK if that comes across a bit better.